### ISSUE TITLE
Cannot compile application with ts-result

### ISSUE THREAD

**Issue by akosmenyhert** [vultix/ts-results#6]
I installed the latest `ts-result` (v2.0.1) with npm and when I try to compile my typescript program I receive a following error: 

Windows 10
Latest typescript: 3.4.1

```
node_modules\ts-results\index.js:1
(function (exports, require, module, __filename, __dirname) { export class Err {
                                                                                              ^^^^^^

SyntaxError: Unexpected token export
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (....\dist\configuration\ConfigurationProcessor.js:13:22)
```

**Comment by Grummfy** [comment]
got the same. Any solution?

**Comment by GabrielCTroia** [comment]
I think I have a fix with PR #7, but it needs to be republished to npm before we can use it.

**Comment by vultix** [comment]
Should be fixed in `ts-results 3.0.0`
